index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="./styles.css">
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">DR. John Maccarthur</h1>
+      <p id="more-info">The man who reached a billion lives</p>
+      <div id="photo-area">
```

Comment:
Use semantic tags instead of reusing divs

Reply:
- This is a 'Yes' for me. Just don't make your comment look like you are giving orders and I think that will make your review comment perfect :ok_hand:

Reply:
- This is a 'Yes' for me. Just don't make your comment look like you are giving orders and I think that will make your review comment perfect :ok_hand: